packages/flutter/lib/src/painting/decoration_image.dart

```diff
@@ -405,6 +405,59 @@ void debugFlushLastFrameImageSizeInfo() {
   }());
 }
 
+/// Information that describes how to tile an image for a given [ImageRepeat]
+/// enum.
+@visibleForTesting
+@immutable
+class DebugImageTilingInfo {
```

Comment:
Probably shouldn't have Debug in the name since this is used in regular release builds?